src/Symfony/Component/Notifier/Bridge/MessageBird/MessageBirdTransport.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MessageBird;
+
+use Symfony\Component\HttpFoundation\Response;
```

Comment:
HttpFoundation is not required in composer.json. 

I suggest we dont import this library just to use a constant for a HTTP status code.

Reply:
Hey, Do you mean this line?

```
        "symfony/http-client": "^4.4|^5.2",
```

Reply:
No. I meant that this line should be removed. It is not needed anymore since you are not using constants from this class.

Reply:
Ok. Removed and changed to 201 :)